src/client/components/Button/Button.js

```diff
@@ -2,9 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './Button.styles.css';
 
-const Button = ({ buttonName, style }) => {
+const Button = ({ buttonName, style, type, onClick }) => {
+  const button = type;
```

Comment:
Why do you declare this instead of just using the `type` prop directly?

Reply:
If i use directly type as a value it throws me an error. so that I declared a variable for that.

Reply:
What error is that?

Reply:
`type` is an invalid value for button type attribute

Reply:
Hm, maybe you can try any of the suggestions listed in this thread? https://github.com/yannickcr/eslint-plugin-react/issues/1555

Reply:
I tried with this before, but nothing was working . If i add es-lint disable and enable comments before and after it works. I am not sure whether it is a right solution to this, but I fixed it in that way.